lib/db/models/Pair.ts

```diff
@@ -19,8 +18,8 @@ class Pair extends Model {
       timestamps: false,
     });
 
-    Pair.beforeBulkCreate(pairs => pairs.forEach(pair => pair.id = getPairId(pair)));
-    Pair.beforeCreate((pair) => { pair.id = getPairId(pair); });
+    Pair.beforeBulkCreate(pairs => pairs.forEach(pair => pair.id = `${pair.base}/${pair.quote}`));
+    Pair.beforeCreate((pair) => { pair.id = `${pair.base}/${pair.quote}`; });
```

Comment:
That is a very blunt way of fixing the circular import. What about moving the `getSuccessfulTrades` method from `Utils.ts` into `CommandHandler.ts` and exporting it there?